packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/vue/next-steps.md

```diff
@@ -0,0 +1,7 @@
+After you've completed a basic-sign flow Vue.js app with the Sign-In Widget, you can extend your app's sign-in experience by adding features such as multifactor or social IdP authentication. There is almost no code required in your app to add these features because the Sign-In Widget provides the mechanism to handle these use cases. These features are enabled by configuring your Okta org and the sign-on policies for your app.
```

Comment:
```suggestion
After you complete a basic-sign flow for the Vue.js app with the Sign-In Widget, you can extend your app's sign-in experience by adding features such as multifactor or social IdP authentication. There is almost no code required in your app to add these features because the Sign-In Widget provides the mechanism to handle these use cases. These features are enabled by configuring your Okta org and the sign-on policies for your app.
```